tests/test_resolver_oai.py

```diff
@@ -10,13 +10,16 @@
 from ocrd_models.utils import extract_mets_from_oai_content
 from ocrd_utils import getLogger, initLogging, LOG_FORMAT
 
+
 @fixture(name="response_dir")
 def fixture_response_dir(tmpdir):
     src = './tests/data/response/oai_get_record_2200909.xml'
-    target_file = str(tmpdir.mkdir('responses').join('oai_get_record_2200909.xml'))
+    target_file = str(tmpdir.mkdir('responses').join(
```

Comment:
Just out of curiosity: Why the short line length? I know PEP8 and tools based on it call for 78 max line length, but I tend to increase that to at least 100-120 because I actually find it more readable if a method doesn't have so many lines. But I have been wondering what kind of problem this might cause.

Reply:
autopep8c might utilize custom configuration files (https://pypi.org/project/autopep8/#configuration), where such parameters can be set. I didn't include one, therefore it just takes the defaults. I don't have a personal favor about row length, it just puzzles me, when I hit formatting shortcut and the file gets literally rewritten :smile:

Reply:
Since we do have a [`pylintrc`](https://github.com/OCR-D/core/blob/master/.pylintrc) (which skips the `line-too-long` check entirely) in place, maybe we could add one for autopep8 and try to find common ground (i.e. argue 😆) about good code style conventions that work for all of us?